integrations/groq/integration.definition.ts

```diff
@@ -6,7 +6,7 @@ import stt from './bp_modules/speech-to-text'
 export default new IntegrationDefinition({
   name: 'groq',
   title: 'Groq',
-  version: '6.4.0',
+  version: '7.0.0',
   readme: 'hub.md',
   icon: 'icon.svg',
   configuration: {
```

Comment:
**chore:** can you remove the `any` config while we're at it ? (you can rm the whole config field)

Reply:
Good catch, done here: https://github.com/botpress/botpress/pull/13404/commits/7f04b2208b60d44bedac3f53904c5ce8c20adfcb